.github/workflows/ansi_sql_mode_test.yml

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: ANSI SQL mode test
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  ansi_sql_test:
+    uses: ./.github/workflows/build_and_test.yml
```

Comment:
oh wow. This is nice. Now we can reuse workflow files. cc @dongjoon-hyun FYI

Reply:
Yes. I suggest that we move all the scheduled workflows to separated files.

Reply:
Yeah, we should separate them. This is nice. @Yikun too FYI

Reply:
Ya, it looks like a nice tip. Thanks, @HyukjinKwon .

Reply:
Cool, really good, also is an inspiration to reuse workflow in downstream repo.

Looks like there are some limitations we can only separated our workflow which is not using `strategy`.

[1] https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations

Reply:
@Yikun the limitation is for the caller. The strategy in "build_and_test.yml" still works.